README.md

```diff
@@ -170,5 +188,13 @@ Write what an example ERB file (aka view) might look like to list all the warrio
 
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
-<!-- code here -->
+<!doctype html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="/css/styles.css">
+  </head>
+  <body>
+    <%= yield %> <!-- load whatever template was called here -->
+  </body>
+</html>
 ```
```

Comment:
This is a good `layout.erb` file, but the question is asking you to create a view file that contains ERB code that iterates through and lists each item in `@warriors`. You could do this with an `each` statement...